### DISCUSSION TITLE
General

### DISCUSSION THREAD

**Discussion by cromatikap** [podzit/CovidGame#5] [General]
Salut @podzit 

Désolé pour l'absence.

Je pense que ce sera plus pratique et mieux organisé si on continue la discussion commencée à #3 plutôt que devoir revenir sur une vieille PR.

-----------

> Bon voilà j'ai fait beaucoup de modifs ces 2 deniers jours, je vais me calmer et te laisser tester la branche dev.
Hésite pas à me faire un retour.

Well done ! Ca rend mieux comme ça :+1: Je pense que tu peux merger la branche `dev` dans `main` maintenant. Par contre il manque le `.htaccess` et le `favicon.ico`, c'est normal ?

**Comment by podzit** [comment]
Il faut qu'on arrête les discussions sur les PR fermées lol
Donc le CSS j'ai corrigé, je pense que tu peux "nettoyer" le code, l'affichage est conforme.
Je vais ajouter dans les projets les idées d'amélioration qu'on a évoqué.

**Reply by cromatikap** [reply]
Tu peux lier les discussions simplement avec un copier/coller de son ID comme ça: #7 

Je te laisse regarder la première PR de l'année:  #8 :100:

**Comment by podzit** [comment]
tout est ok pour moi je merge `dev` dans `main` ! Ça fait une belle v2 pour la nouvelle année !

**Reply by cromatikap** [reply]
Parfait! Je vois c'est en ligne :D

La branche dev est 10 commits derrière, maintenant remerge `main` dans `dev`, ou sinon simplement supprime la branche `dev` et recrée là à partir de `main` comme ça on est sûr de partir avec exactement la même version pour les prochaines fonctionnalités.

**Reply by podzit** [reply]
ok c'est fait !
